### PR TITLE
RM-8738 upgrade Edge to 110

### PR DESCRIPTION
**Pull request by Frazerus**
https://re-motion.atlassian.net/browse/RM-8738
To be merged after selenium upgrade

### CODE REVIEW

File: Remotion/Web/Development.WebTesting/WebDriver/Factories/Edge/EdgeBrowserFactory.cs [hunk 1]
```diff
@@ -55,7 +55,10 @@ public IBrowserSession CreateBrowser (DriverConfiguration configuration)
 
       var extendedEdgeOptions = _edgeConfiguration.CreateEdgeOptions();
       if (configuration.Headless)
-        extendedEdgeOptions.AddArgument("headless");
+        extendedEdgeOptions.AddArgument("headless=new");
+
+      // turn off sidebar which kills some tests.
+      extendedEdgeOptions.AddArgument("disable-features=msHubApps");
```
Comment (MichaelKetting) [inline Remotion/Web/Development.WebTesting/WebDriver/Factories/Edge/EdgeBrowserFactory.cs:61]: Please add this to the docs in the Jira-ticket, too, too

File: Remotion/Web/Development.WebTesting/Web.Development.WebTesting.csproj [hunk 2]
```diff
@@ -22,10 +22,10 @@
     <ProjectReference Include="..\Contracts.DiagnosticMetadata\Web.Contracts.DiagnosticMetadata.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Fork.Coypu" Version="3.2.0.1" />
+    <PackageReference Include="Fork.Coypu" Version="3.3.0" />
     <PackageReference Include="log4net" Version="2.0.15" />
-    <PackageReference Include="Selenium.Support" Version="4.5.0" />
-    <PackageReference Include="Selenium.WebDriver" Version="4.5.0" />
+    <PackageReference Include="Selenium.Support" Version="4.8.1" />
```
Comment (MichaelKetting) [inline Remotion/Web/Development.WebTesting/Web.Development.WebTesting.csproj:27]: Please add the selenium upgrade as a blocking ticket in jira.

**Review by MichaelKetting — APPROVED** [review]

File: Remotion/Web/Development.WebTesting/WebDriver/Factories/Edge/EdgeBrowserFactory.cs (hunk 1 above)
Comment (MichaelKetting) [inline Remotion/Web/Development.WebTesting/WebDriver/Factories/Edge/EdgeBrowserFactory.cs:61]: Do you want to keep the distinct commits for the individual feature toggles? if so, please update the commit messages.

**Review by MichaelKetting — APPROVED** [review]

**Comment by Frazerus** [comment]
Habe noch cases hinzugefügt damit die tests laufen, werde noch ein follow-up ticket erstellen für edge bug.

**Comment by Frazerus** [comment]
https://re-motion.atlassian.net/browse/RM-8767
Created for follow up

File: Remotion/Web/Development.WebTesting.IntegrationTests/FileDownloadTest.cs [hunk 3]
```diff
@@ -113,6 +113,9 @@ public void TestDownloadOpensInNewWindow_WithAnchorTargetBlank ()
       if (Helper.BrowserConfiguration.IsFirefox())
         Assert.Ignore("RM-7856: Test does not work properly after Firefox browser upgrade (Version 89).");
 
+      if(Helper.BrowserConfiguration.IsEdge())
```
Comment (MichaelKetting) [inline Remotion/Web/Development.WebTesting.IntegrationTests/FileDownloadTest.cs:116]:
```suggestion
      if (Helper.BrowserConfiguration.IsEdge())
```

File: Remotion/Web/Development.WebTesting.IntegrationTests/FileDownloadTest.cs [hunk 4]
```diff
@@ -245,6 +248,9 @@ public void TestDownloadOpensInNewWindow_WithPostback_WithXmlFile ()
       if (Helper.BrowserConfiguration.IsFirefox())
         Assert.Ignore("RM-7856: Test does not work properly after Firefox browser upgrade (Version 89).");
 
+
+      if(Helper.BrowserConfiguration.IsEdge())
```
Comment (MichaelKetting) [inline Remotion/Web/Development.WebTesting.IntegrationTests/FileDownloadTest.cs:252]:
```suggestion
      if (Helper.BrowserConfiguration.IsEdge())
```

**Review by MichaelKetting — APPROVED** [review]

**Pull request merged**